packages/css/src/page-menu/page-menu.scss

```diff
@@ -30,6 +30,7 @@
   flex-direction: row;
   flex-wrap: wrap;
   list-style: none;
+  outline-offset: var(--amsterdam-focus-outline-offset);
```

Comment:
This shouldn't be on `amsterdam-page-menu`, right? That's a `ul`. It should be on `amsterdam-page-menu__link` and `.amsterdam-page-menu__button` I think